src/docfx/loc/LocMappingType.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Docs.Build
+{
+    public enum LocMappingType
+    {
+        Repository, // loc files are stored in an independent repository like dotnet/docfx.zh-cn per locale
```

Comment:
use `/// <summary>` comments. 

The examples in the comments does not state clearly which part is _repo_, which part is _folder_.

For the **Folder** convention, is it
```
/zh-cn/articles/a.md
/zh-tw/articles/a.md
/articles/a.md
```
or
```
/localization/zh-cn/articles/a.md
/localization/zh-tw/articles/a.md
articles/a.md
```
The latter feels better when locale list is large.

Suppose we have **Folder** convention, do we still need **SideBySide** convention? We can just support **Folder** for initial release

Reply:
thanks for the suggestion.

For SideBySide, I believe the ideal is from you, where did you get this idea?

---
In reply to: [223997053](https://github.com/dotnet/docfx/pull/3482#discussion_r223997053) [](ancestors = 223997053)

Reply:
It's from here: https://github.com/dotnet/docfx/issues/803

Reply:
we can just support Folder for initial release, like you said

---
In reply to: [224298010](https://github.com/dotnet/docfx/pull/3482#discussion_r224298010) [](ancestors = 224298010)